apps/src/lib/client/tx.rs

```diff
@@ -1198,52 +1184,28 @@ pub async fn submit_tx(
         _ => panic!("Cannot broadcast a dry-run transaction"),
     };
 
-    let websocket_timeout =
-        if let Ok(val) = env::var(ENV_VAR_ANOMA_TENDERMINT_WEBSOCKET_TIMEOUT) {
+    let rpc_timeout =
```

Comment:
nit: could we call this max_wait_time or smth since that's the parameter it's used as? A server side expected timeout rather than a client side enforced timeout. At a glance I interpreted this as the latter.